manifests/rule.pp

```diff
@@ -80,6 +84,22 @@
     fail('Exactly one of "action" or the deprecated "policy" param is required.')
   }
 
+  if $outerface and !($chain in ['FORWARD', 'OUTPUT', 'POSTROUTING']) {
+    fail('Outgoing interface can only be set in the "FORWARD", "OUTPUT" and "POSTROUTING" chains.')
+  } elsif $outerface {
+    $outerface_real = " outerface ${outerface}"
+  } else {
+    $outerface_real = ''
+  }
+
+  if $to_source and $action_temp != 'SNAT' {
+    fail('Setting new source address is only valid with the "SNAT" action.')
+  } elsif $to_source {
+    $to_source_real = " to @ipfilter((${$to_source}))"
```

Comment:
could you remove the leading whitespace here and instead add it in the `$rule = squeeze()` line?